### PR TITLE
SFR-2352: Script to upload digital assets to Google Drive

### PR DESCRIPTION
**Pull request by kylevillegas93**
## Description
- Migrates files from S3 to google drive 

## Testing
`python scripts/migrate_digital_assets.py`

### CODE REVIEW

File: scripts/migrate_digital_assets.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        print(f'Processing file: {file_key}')
+        
+        s3_object = s3.get_object(Bucket=s3_bucket, Key=file_key)
+        file_data = s3_object['Body'].read()
+        
+        try:
+            drive_file_id = upload_to_google_drive(drive_service, file_key, file_data)
+            print(f'Uploaded {file_key} to Google Drive with ID: {drive_file_id}')
+        except Exception as e:
+            print(f'Failed to upload {file_key} to Google Drive: {e}')
```
Comment (mitri-slory) [inline scripts/migrate_digital_assets.py:55]: Do we need the "e" variable here if Exception here will detail the reasons behind the error?

**Review by mitri-slory — APPROVED** [review]
Looks good and I only have one question on a try/except block.

File: scripts/migrate_digital_assets.py (hunk 1 above)
Comment (kylevillegas93) [reply scripts/migrate_digital_assets.py:54]: yes! the print statement here won't know about the exception message by default.

Comment (mitri-slory) [reply scripts/migrate_digital_assets.py:54]: Oh that makes sense. A logger would show the exception message by default not a print statement.

Comment (kylevillegas93) [reply scripts/migrate_digital_assets.py:54]: exactly!

**Pull request merged**